src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/JoinRoom.java

```diff
@@ -21,6 +21,84 @@
 import com.github.splendor_mobile_game.websocket.response.Result;
 
 // TODO: This whole class can be unit tested
+
+/**
+ * Reaction sent when new player joins to already created room.
```

Comment:
Maybe I'm nitpicking, but it should be "...when the player **wants** to join an already...".